definition/Auction.ts

```diff
@@ -47,7 +47,8 @@ type Auction = {
     stackable?: 'yes' | 'no' | 'No' | 'Yes' | 0 | 1,
     distance: number | string,
     notes?: string,
-    bid?: Bid
+    bid?: Bid,
+    tags?: string[],

```

Comment:
```suggestion
    bid?: Bid,
    tags: string[],
```

Since it's defaulted on the structure, can we put the tags as no-optional ?